src/user.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Create a user object based on tokens
+ * @param {Object} tokens
+ * @property {Object} idToken Decoded ID token
+ * @property {Object} accessToken Decoded access token
+ * @returns {Object}
+ */
```

Comment:
The access token should not be needed for constructing the `user` object.  Looks like you are only using it for `authorization`, which we shouldn't commingle.